.github/workflows/tests.yml

```diff
@@ -37,3 +37,21 @@ jobs:
 
       - name: Run unit tests
         run: PYTHONPATH=src python3 -m pytest --color=yes
+
+  debian:
```

Comment:
Meta-level: I wonder if we should take this situation as:

 - a failure to detect incoming changes (ie: we need better/regular testing in this area)
 - an indication that everything actually worked out fine, since we *did* catch the issue relatively quickly...

Reply:
It would have been nice to catch a bit earlier, though. This is more forward-looking.. It's easy and cheap to test on Debian, so let's?